java-driver-archetype/src/main/resources/archetype-resources/src/main/java/Main.java

```diff
@@ -0,0 +1,22 @@
+package ${package};
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.CqlSessionBuilder;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import java.net.InetSocketAddress;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Main {
+  private static final Logger LOG = LoggerFactory.getLogger(Main.class);
+
+  public static void main(String[] args) {
+    CqlSessionBuilder builder = CqlSession.builder();
+    // Set the host and port of the Cassandra server here
+    builder.addContactPoint(InetSocketAddress.createUnresolved("${cassandra-host}", ${cassandra-port}));
```

Comment:
We don't want to give in this archetype a simple configuration mechanism? If we will then we could use an info when user interactively type host and port and load those settings here

Reply:
It will default to `127.0.0.1` and `9042` here, if you don't explicitly set these two properties. The defaults are in the `archetype-metadata.xml` required properties section. So, when you run `archetype:generate`, if you do not specify these properties (i.e. `-Dcassandra-host=someHost`), it uses the defaults.

You have the chance to change them as there is a confirmation prompt that shows the values to be used. If you don't like what is set, you enter the interactive mode where you can type in what you want. I tried to cover it in the README.md, in the `Interactive confirmation` section. Let me know if I should expand on that a bit more.

Reply:
Ok, so I think that this comment `Set the host and port of the Cassandra server here` is a bit misleading. If I will be a user, I would think that we should set them manually but maybe it is only my impression